jenkins/resource_jenkins_credential_secret_file_test.go

```diff
@@ -0,0 +1,173 @@
+package jenkins
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	jenkins "github.com/bndr/gojenkins"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccJenkinsCredentialSecretFile_basic(t *testing.T) {
+	var cred jenkins.FileCredentials
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckJenkinsCredentialSecretFileDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: `
+				resource jenkins_credential_secret_file foo {
+				  name = "test-secret-file"
+				  filename = "secret.txt"             
+				  secretbytes = "VGhpcyBpcyBhIHRlc3Qu"
```

Comment:
I'd allow this, but consider a test strategy more reflective of the examples in the docs.

e.g. use [ t's TempDir](https://pkg.go.dev/testing#T.TempDir) and add a dummy file to it, then use Terraform's `base64encode` on the file contents.

Reply:
done